test/Sintrop.test.js

```diff
@@ -304,6 +309,34 @@ contract("Sintrop", (accounts) => {
   });
 
   context("when activist accept inspection", () => {
+    it("should return error when before timeToExpire", async () => {
```

Comment:
vale ajustar o contexto pra o novo nome da variável iy algo como `should return error when other accepted inspection is not expired yet`. Algo nesse sentido.

O que vc acha?

Reply:
atualizado